README.md

```diff
@@ -180,30 +228,14 @@ After installing:
     --outfile=bundle.js
     ```
 
-3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support. We recommend [`babel-polyfill`](https://babeljs.io/docs/en/babel-polyfill/).
-
-
-## Usage guide
-
-For an overview of the Widget's features and authentication flows, check out [our developer docs](https://developer.okta.com/code/javascript/okta_sign-in_widget). There, you will learn how to use the Widget to:
+3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support. We recommend [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill/).
 
-* Sign In to Okta with the Default Dashboard
-* Sign In to Okta with a Custom Dashboard
-* Sign In to Your Application
+### Usage examples
 
-You can also browse the full [API reference documentation](#api-reference).
+#### OIDC login flow using PKCE (Proof Key for Code Exchange)
```

Comment:
Will there be confusion since PKCE is technically "Authorization Code with PKCE", and we discuss Authorization Code below as if it isn't available for SPAs?